diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9e91617..6913a68 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1057,7 +1057,7 @@ else()
   set(enable_pcre2_32 "no")
 endif()
 
-configure_file(pcre2-config.in pcre2-config @ONLY NEWLINE_STYLE LF)
+#configure_file(pcre2-config.in pcre2-config @ONLY NEWLINE_STYLE LF)
 
 # Executables
 
@@ -1248,43 +1248,43 @@ install(
   ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
 )
 install(FILES ${pkg_config_files} DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
-install(
-  FILES "${CMAKE_CURRENT_BINARY_DIR}/pcre2-config"
-  DESTINATION bin
-  # Set 0755 permissions
-  PERMISSIONS OWNER_WRITE OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
-)
+#install(
+#  FILES "${CMAKE_CURRENT_BINARY_DIR}/pcre2-config"
+#  DESTINATION bin
+#  # Set 0755 permissions
+#  PERMISSIONS OWNER_WRITE OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
+#)
 
 install(FILES ${PCRE2_HEADERS} ${PCRE2POSIX_HEADERS} DESTINATION include)
 
 # CMake config files.
-set(PCRE2_CONFIG_IN ${CMAKE_CURRENT_SOURCE_DIR}/cmake/pcre2-config.cmake.in)
-set(PCRE2_CONFIG_OUT ${CMAKE_CURRENT_BINARY_DIR}/cmake/pcre2-config.cmake)
-configure_file(${PCRE2_CONFIG_IN} ${PCRE2_CONFIG_OUT} @ONLY)
-set(PCRE2_CONFIG_VERSION_IN ${CMAKE_CURRENT_SOURCE_DIR}/cmake/pcre2-config-version.cmake.in)
-set(PCRE2_CONFIG_VERSION_OUT ${CMAKE_CURRENT_BINARY_DIR}/cmake/pcre2-config-version.cmake)
-configure_file(${PCRE2_CONFIG_VERSION_IN} ${PCRE2_CONFIG_VERSION_OUT} @ONLY)
-install(FILES ${PCRE2_CONFIG_OUT} ${PCRE2_CONFIG_VERSION_OUT} DESTINATION "${PCRE2_INSTALL_CMAKEDIR}")
-
-file(GLOB html ${PROJECT_SOURCE_DIR}/doc/html/*.html ${PROJECT_SOURCE_DIR}/doc/html/*.txt)
-file(
-  GLOB txts
-  ${PROJECT_SOURCE_DIR}/doc/*.txt
-  AUTHORS.md
-  COPYING
-  ChangeLog
-  LICENCE.md
-  NEWS
-  README
-  SECURITY.md
-)
-file(GLOB man1 ${PROJECT_SOURCE_DIR}/doc/*.1)
-file(GLOB man3 ${PROJECT_SOURCE_DIR}/doc/*.3)
-
-install(FILES ${man1} DESTINATION ${CMAKE_INSTALL_MANDIR}/man1)
-install(FILES ${man3} DESTINATION ${CMAKE_INSTALL_MANDIR}/man3)
-install(FILES ${txts} DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/doc/pcre2)
-install(FILES ${html} DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/doc/pcre2/html)
+#set(PCRE2_CONFIG_IN ${CMAKE_CURRENT_SOURCE_DIR}/cmake/pcre2-config.cmake.in)
+#set(PCRE2_CONFIG_OUT ${CMAKE_CURRENT_BINARY_DIR}/cmake/pcre2-config.cmake)
+#configure_file(${PCRE2_CONFIG_IN} ${PCRE2_CONFIG_OUT} @ONLY)
+#set(PCRE2_CONFIG_VERSION_IN ${CMAKE_CURRENT_SOURCE_DIR}/cmake/pcre2-config-version.cmake.in)
+#set(PCRE2_CONFIG_VERSION_OUT ${CMAKE_CURRENT_BINARY_DIR}/cmake/pcre2-config-version.cmake)
+#configure_file(${PCRE2_CONFIG_VERSION_IN} ${PCRE2_CONFIG_VERSION_OUT} @ONLY)
+#install(FILES ${PCRE2_CONFIG_OUT} ${PCRE2_CONFIG_VERSION_OUT} DESTINATION "${PCRE2_INSTALL_CMAKEDIR}")
+
+#file(GLOB html ${PROJECT_SOURCE_DIR}/doc/html/*.html ${PROJECT_SOURCE_DIR}/doc/html/*.txt)
+#file(
+#  GLOB txts
+#  ${PROJECT_SOURCE_DIR}/doc/*.txt
+#  AUTHORS.md
+#  COPYING
+#  ChangeLog
+#  LICENCE.md
+#  NEWS
+#  README
+#  SECURITY.md
+#)
+#file(GLOB man1 ${PROJECT_SOURCE_DIR}/doc/*.1)
+#file(GLOB man3 ${PROJECT_SOURCE_DIR}/doc/*.3)
+
+#install(FILES ${man1} DESTINATION ${CMAKE_INSTALL_MANDIR}/man1)
+#install(FILES ${man3} DESTINATION ${CMAKE_INSTALL_MANDIR}/man3)
+#install(FILES ${txts} DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/doc/pcre2)
+#install(FILES ${html} DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/doc/pcre2/html)
 
 if(MSVC AND INSTALL_MSVC_PDB)
   install(FILES ${DLL_PDB_FILES} DESTINATION bin CONFIGURATIONS RelWithDebInfo)
