diff --git a/ffbuild/pkgconfig_generate.sh b/ffbuild/pkgconfig_generate.sh
index e5de6716d2..b04ff68188 100755
--- a/ffbuild/pkgconfig_generate.sh
+++ b/ffbuild/pkgconfig_generate.sh
@@ -27,10 +27,10 @@ requires=${requires%, }
 version=$(grep ${name}_VERSION= $name/${name}.version | cut -d= -f2)
 
 cat <<EOF > $name/$fullname.pc
-prefix=$prefix
+prefix=\${pcfiledir}/../..
 exec_prefix=\${prefix}
-libdir=$libdir
-includedir=$incdir
+libdir=\${prefix}/lib
+includedir=\${prefix}/include
 
 Name: $fullname
 Description: $comment
