diff --git a/libavcodec/audiotoolboxdec.c b/libavcodec/audiotoolboxdec.c
index 0f7ce8e4eb..a0fc27f76c 100644
--- a/libavcodec/audiotoolboxdec.c
+++ b/libavcodec/audiotoolboxdec.c
@@ -36,7 +36,7 @@
 #include "libavutil/opt.h"
 #include "libavutil/log.h"
 
-#if __MAC_OS_X_VERSION_MIN_REQUIRED < 101100
+#if MAC_OS_X_VERSION_MIN_REQUIRED < 101100
 #define kAudioFormatEnhancedAC3 'ec-3'
 #endif
 
diff --git a/libavcodec/pcm-dvdenc.c b/libavcodec/pcm-dvdenc.c
index 1e7ee644f6..2da29a9abb 100644
--- a/libavcodec/pcm-dvdenc.c
+++ b/libavcodec/pcm-dvdenc.c
@@ -35,7 +35,7 @@ typedef struct PCMDVDContext {
 static av_cold int pcm_dvd_encode_init(AVCodecContext *avctx)
 {
     PCMDVDContext *s = avctx->priv_data;
-    int quant, freq, frame_size;
+    int quant = 0, freq, frame_size;
 
     switch (avctx->sample_rate) {
     case 48000:
