diff --git a/configure b/configure
index 00c3cee33f..87a8fffd32 100755
--- a/configure
+++ b/configure
@@ -7386,7 +7386,7 @@ fi
 enabled debug && add_cflags -g"$debuglevel" && add_asflags -g"$debuglevel"

 # add some useful compiler flags if supported
-check_cflags -Wdeclaration-after-statement
+# check_cflags -Wdeclaration-after-statement
 check_cflags -Wall
 check_cflags -Wdisabled-optimization
 check_cflags -Wpointer-arith
