diff --git a/expat/CMakeLists.txt b/expat/CMakeLists.txt
index 1f650339..ab591f5a 100644
--- a/expat/CMakeLists.txt
+++ b/expat/CMakeLists.txt
@@ -835,13 +835,13 @@ endif()
 #
 # Documentation
 #
-configure_file(Changes changelog COPYONLY)
-expat_install(
-    FILES
-        AUTHORS
-        ${CMAKE_CURRENT_BINARY_DIR}/changelog
-    DESTINATION
-        ${CMAKE_INSTALL_DOCDIR})
+#configure_file(Changes changelog COPYONLY)
+#expat_install(
+#    FILES
+#        AUTHORS
+#        ${CMAKE_CURRENT_BINARY_DIR}/changelog
+#    DESTINATION
+#        ${CMAKE_INSTALL_DOCDIR})
 
 #
 # CMake files for find_package(expat [..] CONFIG [..])
