diff --git a/build/cmake/install.cmake b/build/cmake/install.cmake
index 384c6837b8..25c15a225b 100644
--- a/build/cmake/install.cmake
+++ b/build/cmake/install.cmake
@@ -31,21 +31,21 @@ else()
         DIRECTORY "${wxSETUP_HEADER_PATH}"
         DESTINATION "lib/wx/include")
 
-    install(
-        FILES "${wxOUTPUT_DIR}/wx/config/${wxBUILD_FILE_ID}"
-        DESTINATION "lib/wx/config"
-        PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ
-                    GROUP_EXECUTE GROUP_READ
-                    WORLD_EXECUTE WORLD_READ
-        )
+    #install(
+    #    FILES "${wxOUTPUT_DIR}/wx/config/${wxBUILD_FILE_ID}"
+    #    DESTINATION "lib/wx/config"
+    #    PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ
+    #                GROUP_EXECUTE GROUP_READ
+    #                WORLD_EXECUTE WORLD_READ
+    #    )
 
-    install(DIRECTORY DESTINATION "bin")
-    install(CODE "execute_process( \
-        COMMAND ${CMAKE_COMMAND} -E create_symlink \
-        \"${CMAKE_INSTALL_PREFIX}/lib/wx/config/${wxBUILD_FILE_ID}\" \
-        \"\$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}/bin/wx-config\" \
-        )"
-    )
+    #install(DIRECTORY DESTINATION "bin")
+    #install(CODE "execute_process( \
+    #    COMMAND ${CMAKE_COMMAND} -E create_symlink \
+    #    \"${CMAKE_INSTALL_PREFIX}/lib/wx/config/${wxBUILD_FILE_ID}\" \
+    #    \"\$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}/bin/wx-config\" \
+    #    )"
+    #)
 endif()
 
 install(EXPORT wxWidgetsTargets NAMESPACE wx:: DESTINATION "lib/cmake/wxWidgets/${wxPLATFORM_LIB_DIR}")
diff --git a/build/cmake/utils/CMakeLists.txt b/build/cmake/utils/CMakeLists.txt
index dbed8cc9b3..c937b90c10 100644
--- a/build/cmake/utils/CMakeLists.txt
+++ b/build/cmake/utils/CMakeLists.txt
@@ -16,10 +16,10 @@ if(wxUSE_XRC)
     wx_exe_link_libraries(wxrc wxbase)
 
     set(wxrc_output_name "wxrc")
-    if(NOT WIN32_MSVC_NAMING)
-        wx_get_flavour(lib_flavour "-")
-        set(wxrc_output_name "wxrc-${wxMAJOR_VERSION}.${wxMINOR_VERSION}${lib_flavour}")
-    endif()
+    #if(NOT WIN32_MSVC_NAMING)
+    #    wx_get_flavour(lib_flavour "-")
+    #    set(wxrc_output_name "wxrc-${wxMAJOR_VERSION}.${wxMINOR_VERSION}${lib_flavour}")
+    #endif()
 
     set_target_properties(wxrc PROPERTIES OUTPUT_NAME ${wxrc_output_name})
 
@@ -38,12 +38,12 @@ if(wxUSE_XRC)
         endif()
 
         # Don't use wx_install() here to preserve escaping.
-        install(CODE "execute_process( \
-            COMMAND ${CMAKE_COMMAND} -E create_symlink \
-            \"${CMAKE_INSTALL_PREFIX}/bin/${wxrc_output_name}${EXE_SUFFIX}\" \
-            \"\$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}/bin/wxrc${EXE_SUFFIX}\" \
-            )"
-        )
+        #install(CODE "execute_process( \
+        #    COMMAND ${CMAKE_COMMAND} -E create_symlink \
+        #    \"${CMAKE_INSTALL_PREFIX}/bin/${wxrc_output_name}${EXE_SUFFIX}\" \
+        #    \"\$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}/bin/wxrc${EXE_SUFFIX}\" \
+        #    )"
+        #)
     endif()
 endif()
 
